x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_list/components/alerts_list.test.tsx

```diff
@@ -127,116 +127,148 @@ describe('alerts_list component empty', () => {
 
     wrapper.find('button[data-test-subj="createFirstAlertButton"]').simulate('click');
 
-    // When the AlertAdd component is rendered, it waits for the healthcheck to resolve
-    await new Promise((resolve) => {
-      setTimeout(resolve, 1000);
+    await act(async () => {
+      // When the AlertAdd component is rendered, it waits for the healthcheck to resolve
+      await new Promise((resolve) => {
+        setTimeout(resolve, 1000);
```

Comment:
potential flakiness here, in the future?

Reply:
The `setTimeout` has been part of this test for a while and seems to be holding up. I noticed a warning to wrap calls that cause a re-render inside an `act` while running the tests so that's what I did here.